pkg/vmprovider/providers/vsphere2/vmlifecycle/bootstrap.go

```diff
@@ -61,20 +61,25 @@ func DoBootstrap(
 	bootstrap := vmCtx.VM.Spec.Bootstrap
 	if bootstrap == nil {
 		// For backwards compat we have to keep going and default to LinuxPrep.
```

Comment:
Can we use some type of pattern that indicates when something is for a specific schema version? Maybe:

```go
V1ALPHA1: Default to LinuxPrep
```